static/app/views/alerts/rules/details/body.tsx

```diff
@@ -290,13 +291,6 @@ export default class DetailsBody extends React.Component<Props> {
         {({initiallyLoaded, projects}) => {
           return initiallyLoaded ? (
             <React.Fragment>
-              <StyledLayoutBody>
-                <StyledAlert type="info" icon={<IconInfo size="md" />}>
-                  {t(
-                    'You’re viewing the new alert details page. To view the old experience, select an alert on the chart or in the history.'
-                  )}
-                </StyledAlert>
-              </StyledLayoutBody>
```

Comment:
We no longer need this alert since selecting an incident on the chart zooms in now.